### DISCUSSION TITLE
Traceback Error Resulting from Deleting a Dag

### DISCUSSION THREAD

**Discussion by ghost** [apache/airflow#17845] [Q&A]
Python version: 3.8.10
Airflow version: 2.1.3

Hello,

Whenever I try and delete a DAG, I receive a Traceback Error. I have tried to change load_examples = False inside airflow.cfg and reset the airflow db and still have no luck removing the sample DAGs. Any solutions?

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
This is not an issue with airflow - it's the way you are running it. Please next time open discussion instead of issue if you do not have reproducible path, but you want to get help with your way of using airflow.
 
You need to make sure you restart airflow components after you change the configuration and make sure that your load_examples setting is correctly read by airflow (you can double check it by running `airflow info` `airflow config`.
